portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -16,17 +16,53 @@
 
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
+import java.util.*;  
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    private String name = "Jordan";
```

Comment:
This looks like a constant. WDYT about making it `static final` and naming it `NAME`? (conventionally constants are named in all caps snake case LIKE_THIS)

Reply:
i actually think that thi sname value was just for the intial testing of the portfolio, I beleive moving forward to Step 5 of Week 3 I will remove this constant